terraform/projects/app-ecs-instances/instance-user-data.tpl

```diff
@@ -32,10 +32,16 @@ else
   echo "[$(date '+%H:%M:%S %d-%m-%Y')] attach-volume: /dev/$DEVICE is already formatted: $(file -s /dev/$DEVICE)"
 fi
 
-# Allow prometheus container user access to read/write/execute within container
+# Allow prometheus container user access to read/write metrics to persistent volume mount from within a container
```

Comment:
It might be a little clearer if the comment on this line was something like:
`Mount volume to be used by prometheus container`

And the comment on the second section was like:
` Create prometheus group and allow it to read and write to our volume for storing prometheus data. Note, 65534 is chosen as the UID to be added to the prometheus group as this is the UID that prometheus in the docker container runs as.`

My point regarding a link specifying the UID was that someone looking at this code probably won't know where the number 65534 has come from. They could find out by going into the docker container as you demonstrated but if they could just read it in some external docs for prometheus that would be quicker/easier. However I don't think the prometheus docs document this fact (the best I could find was github issues of people complaining about the choice of UID and referenced it there).

Reply:
True, unfortunately we don't have any form of documentation around this. This is just one of those things. 

We will have to assume that the comments & the code will be enough for them. Let us also hope that they stumble upon these comments.